interface/src/app/node/node.js

```diff
@@ -5,7 +5,10 @@
 angular.module('adage.node', [
   'ui.router',
   'ui.bootstrap',
-  'ngResource'
+  'ngResource',
+  // Dependency modules from the 'adage' Angular app
+  'adage.utils'
```

Comment:
Huh. I thought there was other stuff in there, but all I see is the `MathFuncts` factory that @rzelayafavila just added. I must have been thinking of something else. I would advocate for changing the name from `adage.utils` to `adage.stats` or even `greenelab.stats` if we think this will be used outside of the `adage-server`.